xain_fl/coordinator/metrics_store.py

```diff
@@ -91,8 +82,6 @@ def write_metrics(self, participant_id: str, metrics: Dict[str, ndarray]) -> boo
         except Exception as err:  # pylint: disable=broad-except
```

Comment:
Yeah I guess the influx db client raises custom exceptions. cc @Robert-Steiner

Reply:
Right the `InfluxDBClient` raises custom exceptions like `InfluxDBClientError` and `InfluxDBServerError`. The issue is, that the InfluxDB client doesn't catch exceptions and wrap them into a `InfluxDBClientError` internally. So that means, that the call `self.influx_client.write_points(influx_data_points)` will raise a `HTTPError` from the `urllib` package instead of raising a `InfluxDBClientError`.

Reply:
Then maybe we could catch the three of them to be sure not to miss one: `InfluxDBClientError`, `InfluxDBServerError`, and `HTTPError`.

Reply:
The HTTPError was just an example, the problem is that it [isn't documented](https://influxdb-python.readthedocs.io/en/latest/exceptions.html) which exception could be raised.

Reply:
Ah ok. That's not pretty... I guess we could catch everything _but_ `KeyboardInterrupt` then. Would you be happier with that @finiteprods ? :smile: